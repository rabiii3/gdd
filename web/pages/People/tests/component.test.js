import React from 'react';
import { shallow } from 'enzyme';
import { Page } from '..';
import { IntlProvider } from 'react-intl';

describe('web | pages | People | index', () => {
  describe('render', () => {
    it('should match snapshot', () => {
      const intlProvider = new IntlProvider({ locale: 'en', messages: {} }, {});
      const { intl } = intlProvider.getChildContext();
      const props = {
        classes: {},
        people: [],
        actions: {},
        term: '',
        intl: { formatMessage: jest.fn() },
      };

      const wrapper = shallow(<Page {...props} intl={intl} />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
