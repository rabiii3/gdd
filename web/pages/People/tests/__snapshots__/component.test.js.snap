// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`web | pages | People | index render should match snapshot 1`] = `
<WithStyles(Container)>
  <WithStyles(StyledHeader)>
    <WithStyles(StyledHeaderLeft)>
      <WithStyles(ResumeIcon) />
      <Spacer
        size="MEDIUM"
      />
      <WithStyles(Title)
        text="People Page"
      />
    </WithStyles(StyledHeaderLeft)>
    <WithStyles(StyledHeaderRight)>
      <Hidden
        implementation="js"
        lgDown={false}
        lgUp={false}
        mdDown={false}
        mdUp={false}
        smDown={false}
        smUp={false}
        xlDown={false}
        xlUp={false}
        xsDown={true}
        xsUp={false}
      >
        <WithStyles(SearchBar)
          onCancelSearch={[Function]}
          onChange={[Function]}
        />
      </Hidden>
    </WithStyles(StyledHeaderRight)>
  </WithStyles(StyledHeader)>
  <WithStyles(Content)>
    <WithStyles(Component)
      actions={Object {}}
      people={Array []}
    />
  </WithStyles(Content)>
</WithStyles(Container)>
`;
